core/src/apps/bitcoin/common.py

```diff
@@ -102,6 +107,11 @@ def from_int(cls, sighash_type: int) -> "SigHashType":
     InputScriptType.SPENDMULTISIG,
 )
 
+LEAF_VERSION = const(0xC0)
+GENERATOR = const(
```

Comment:
maybe let's add a comment that the generator value is taken from https://github.com/bitcoin/bips/blob/master/bip-0341.mediawiki#constructing-and-spending-taproot-outputs

Reply:
I will actually update this to instead use a BIP32 node with this as the pubkey and a chaincode of 32 `0` bytes, and then we can derive the unspendable internal pubkey for the multisig path. That will make this compatible with BIP388 wallet policies, which require the all keys to be a xpubs and use the same derivation path.

Reply:
That way other wallets can use the same construction and have the xpub for the internalPubkey be `xpub661MyMwAqRbcEYS8w7XLSVeEsBXy79zSzH1J8vCdxAZningWLdN3zgtU6QgnecKFpJFPpdzxKrwoaZoV44qAJewsc4kX9vGaCaBExuvJH57`, as described [here](https://delvingbitcoin.org/t/unspendable-keys-in-descriptors/304).

Reply:
@prusnak added more commentary [here](https://github.com/trezor/trezor-firmware/pull/4159/files#diff-c528ee9407682668ced569943303086cb4098504c4a3684aa85b2f766b283763R85-R105).